docs/markdown/ios/card/subcomponents/card.boards.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Card.Boards
+description: The Boards preview block or “Boards PB” utilizes two of the Board and Section components to present a collection of imagery that typically represents a Card.Boards.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/76/6a/bf/766abf51e7086e0e7b2dddefc7e94f75.png" alt="Example of a Card container with a Card.Boards preview that contains two Card.Boards representations." noPadding color="background-elevation-accent"/>
+
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
+- To suggest boards containing content that a Pinner might like based on their engagement, interests, boards, demographic information, or other signals
+- To suggest boards created by another user (ex: Pinner, company, brand)
+- To suggest boards featuring a Pin(s) the user has saved/looked at
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- When showing any content that is not contained within a Card.Boards
+</Group>
+</TwoCol>
+
+## Best practices
+
+- Always show two boards or sections per card. More or less than two will display inconsistently within the card. 
```

Comment:
Are we not using cards here intentionally?

Reply:
Yes. I think after this, we need to regroup on the design side and figure out if we want to do this or not. It feels inconsistent, but that's how these were written.